app/views/catalog/_show_default_attribute_table.html.erb

```diff
@@ -1,19 +1 @@
-<% if show_attribute_table? %>
```

Comment:
do we want to preserve this check? If so, I think all you need to do is add a `render?` to `AttributeTableComponent` that calls it